README.md

```diff
@@ -0,0 +1,30 @@
+Chameleon Shop
+==============
+
+This is the shop repository of the Chameleon System, providing shop-related functionality. This repository should be
+installed and used in conjunction with the [Chameleon System](https://github.com/chameleon-system/chameleon-system).
+See that repository for details.
+
+Documentation
+-------------
+
+Please see the [Chameleon System main repository](https://github.com/chameleon-system/chameleon-system) for details on
+documentation.
+
+Installation
+------------
+
+Please see the [Chameleon System main repository](https://github.com/chameleon-system/chameleon-system) for an
+installation guide.
+
+License
+-------
+
+This repository is licensed under the MIT License. See the complete license under `LICENSE` for details.
+
+Reporting an Issue or a Feature Request
+---------------------------------------
+
+Please report issues and feature requests at the official GitHub repository of the [Chameleon System main
```

Comment:
```suggestion
Please report issues and feature requests at the chameleon-system repository of the [Chameleon System main
```

Reply:
See comment in chameleon-base - I think the original sentence suits better.

Reply:
Changed it to read the same as in chameleon-base.